site/public/css/rainbow-customization.css

```diff
@@ -223,6 +223,10 @@ hr {
     padding: 4px 6px;
 }
 
+.note-input {
+    width: 32.5em !important;
```

Comment:
Is there a way to do this without `!important`?  Using `!important` can be challenging from a future maintenance perspective since it is difficult to override and should be avoided if at all possible.

Reply:
An ID should be unique on the page, while a class can appear many times.  In reality, most browsers can handle non-unique IDs, but their behavior in such cases is not defined.  The most common case where this comes up is when testing with Cypress.

Is there a reason why an ID works but a class doesn't?

Reply:
More specific types of selectors override in CSS, and id counts as more specific than class.